internal/service/ipam/model_ipamsvc_range.go

```diff
@@ -0,0 +1,280 @@
+	"created_at":                 timetypes.RFC3339Type{},
+	"dhcp_host":                  types.StringType,
+	"dhcp_options":               types.ListType{ElemType: types.ObjectType{AttrTypes: IpamsvcOptionItemAttrTypes}},
+	"disable_dhcp":               types.BoolType,
+	"end":                        types.StringType,
+	"exclusion_ranges":           types.ListType{ElemType: types.ObjectType{AttrTypes: IpamsvcExclusionRangeAttrTypes}},
+	"filters":                    types.ListType{ElemType: types.ObjectType{AttrTypes: IpamsvcAccessFilterAttrTypes}},
+	"id":                         types.StringType,
+	"inheritance_assigned_hosts": types.ListType{ElemType: types.ObjectType{AttrTypes: InheritanceAssignedHostAttrTypes}},
+	"inheritance_parent":         types.StringType,
+	"inheritance_sources":        types.ObjectType{AttrTypes: IpamsvcDHCPOptionsInheritanceAttrTypes},
+	"name":                       types.StringType,
+	"parent":                     types.StringType,
+	"protocol":                   types.StringType,
+	"space":                      types.StringType,
+	"start":                      types.StringType,
+	"tags":                       types.MapType{ElemType: types.StringType},
+	"threshold":                  types.ObjectType{AttrTypes: IpamsvcUtilizationThresholdAttrTypes},
+	"updated_at":                 timetypes.RFC3339Type{},
+	"utilization":                types.ObjectType{AttrTypes: IpamsvcUtilizationAttrTypes},
+	"utilization_v6":             types.ObjectType{AttrTypes: IpamsvcUtilizationV6AttrTypes},
+}
+
+var IpamsvcRangeResourceSchemaAttributes = map[string]schema.Attribute{
+	"comment": schema.StringAttribute{
+		Optional: true,
+		Validators: []validator.String{
+			stringvalidator.LengthBetween(0, 1024),
+		},
+		MarkdownDescription: "The description for the range. May contain 0 to 1024 characters. Can include UTF-8.",
+	},
+	"created_at": schema.StringAttribute{
+		CustomType:          timetypes.RFC3339Type{},
+		Computed:            true,
+		MarkdownDescription: "Time when the object has been created.",
+	},
+	"dhcp_host": schema.StringAttribute{
+		Optional:            true,
+		MarkdownDescription: "The resource identifier.",
+	},
+	"dhcp_options": schema.ListNestedAttribute{
+		NestedObject: schema.NestedAttributeObject{
+			Attributes: IpamsvcOptionItemResourceSchemaAttributes,
+		},
+		Optional:            true,
+		MarkdownDescription: "The list of DHCP options. May be either a specific option or a group of options.",
+	},
+	"disable_dhcp": schema.BoolAttribute{
+		Optional:            true,
+		Computed:            true,
+		Default:             booldefault.StaticBool(false),
+		MarkdownDescription: "Optional. _true_ to disable object. A disabled object is effectively non-existent when generating configuration.  Defaults to _false_.",
+	},
+	"end": schema.StringAttribute{
+		Required: true,
+		PlanModifiers: []planmodifier.String{
+			stringplanmodifier.RequiresReplaceIfConfigured(),
+		},
```

Comment:
You don't need this. API allows you to update `end`

Reply:
True. I should have removed it. Will clean it up